procedures/MT/rotator/rotator_verification.ipynb

```diff
@@ -0,0 +1,1825 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        Slew supersession test.
</pre><p>what is supersession? I've not heard that term..</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Improved the document in commit: [da2822c](https://github.com/lsst-ts/ts_notebooks/pull/96/commits/da2822c02b133ed099461c12c208f882ed94bd6d). Thanks!